netmap/netmap_contract.go

```diff
@@ -73,7 +74,8 @@ const (
 	containerContractKey = "containerScriptHash"
```

Comment:
We may add `nolint` comment, we need to keep our linter job with OK status.

Reply:
oh, i think i got it. it is used only in `_deploy` func but this func is nolinted already and is ignored by the linter and the linter does not see its usage

added `nolint`